src/components/signupform/signupform.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from "react";
+import { Form, Button } from "react-bootstrap";
+import "./signupform.scss";
+
+class SignUpForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      isLogin: true,
+      email: "",
+      password: "",
+      passwordVerify: "",
+    };
+  }
+
+  handleChange = (params, event) => {
+    event.preventDefault();
+    if (params === "email") {
+      this.setState({ email: event.target.value });
+    } else if (params === "password") {
+      this.setState({ password: event.target.value });
+    } else if (params === "passwordVerify") {
+      this.setState({ passwordVerify: event.target.value });
+    } else if (params === "username") {
+      this.setState({ username: event.target.value });
+    }
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    if (this.state.password !== this.state.passwordVerify) {
+      alert("Password does not match with Re-Entered Password!");
+      return;
+    } else {
+      console.log(this.state.username);
+      let requestBody = {
+        query: `
+          mutation {
+            createUser(userInput:{
+              email:"${this.state.email}",
+              password:"${this.state.password}",
+              username: "${this.state.username}"
```

Comment:
I think maybe we want to use back ticks instead of quotes (`" "`) here.

Reply:
Backticks do not work inside the backticks

Reply:
@DevanshCodes please dont resolve comments ask the reviewer to do the same.